docs/en/index.md

```diff
@@ -11,117 +11,117 @@ Last released to Markdown: not yet released
 -->
 
 
-## Intro to Ubuntu IoT Developer documentation
+## Introduction to Ubuntu IoT Developer documentation
 Ubuntu IoT Developer documentation covers the core set of platform and service
 technologies that make up Ubuntu’s Internet of Things (IoT) offering. It 
-includes a set of Ubuntu operating systems geared towards different device 
+includes a set of Ubuntu operating systems (OSes) geared towards different device 
 use cases and associated cloud services. 
 
-Classic Ubuntu includes the traditional Ubuntu Desktop, Ubuntu Server and 
+Ubuntu Classic includes the traditional Ubuntu Desktop, Ubuntu Server and 
```

Comment:
This is tricky. We are making lots of decisions about what is and is not a product, some of which are pretty vague/arbitrary. I like the feel of Ubuntu Core/Ubuntu Classic much more than Ubuntu Core/classic Ubuntu for reasons of balance and parity. I could argue  that Ubuntu Classic is a product that it has two flavors: Ubuntu Desktop and Ubuntu Server.

Reply:
I'm not making this up, we literally don't have a proper noun Ubuntu Classic - it's classic Ubuntu

Reply:
I disagree and think it is a matter of interpretation. "Classic" is capitalized for example in Seeding a Classic image doc. I'd like to hear Nick's view. If he agrees with you, that is fine.

Reply:
Sorry, you're misunderstanding my point here - this isn't my interpretation, this comes from the marketing team...we're just not allowed to refer to Ubuntu Classic as a proper noun because there is no such product name.